cmd/tpl/main.go

```diff
@@ -91,13 +91,15 @@ func main() {
 	})
 	if vaultPrefix != "" {
 		w.Vault().Prefix = vaultPrefix
+		w.Azure().Prefix = vaultPrefix
```

Comment:
Please do not reuse the keymapping and previous of Vault for Azure KeyVault but use separate flags.